### PR TITLE
Add instructions for submitting detailed logs

### PR DESCRIPTION
**Pull request by chwarr**
Resolves https://github.com/Microsoft/WSL/issues/3137

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -121,3 +124,9 @@ lxcore_kernel.etl
 lxcore_service.etl
 lxcore_user.etl
 ```
+
+#### Submitting logs
```
Comment (benhillis) [inline CONTRIBUTING.md]: The ETL files do not contain sensitive information.

Comment (chwarr) [reply CONTRIBUTING.md]: Cool. I'll revise. (This was part of the uncertainty I had, hence the reason I opened #3137.)

**Comment by chwarr** [comment]
Checking back in after a couple of days. Are more changes needed? Anything else that's blocking this from being merged?

**Comment by benhillis** [comment]
@chwarr - Other than my vacation, nope! Thanks for taking care of this.

**Comment by Biswa96** [comment]
Is it possible to collect ETL with `tracelog.exe` insead of `logman.exe`?

**Pull request merged**